src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,145 @@
+import static org.mockito.Mockito.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.dao.MenuProductDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+
+	@Mock
+	private MenuDao menuDao;
+	@Mock
+	private MenuGroupDao menuGroupDao;
+	@Mock
+	private MenuProductDao menuProductDao;
+	@Mock
+	private ProductDao productDao;
+	@InjectMocks
+	private MenuService menuService;
+
+	private Menu A세트;
+	private Menu B세트;
+	private MenuProduct 탕수육중;
+	private MenuProduct 깐풍기중;
+	private Product 탕수육;
+	private Product 깐풍기;
+	private MenuGroup 중식;
+	private List<MenuProduct> menuProducts;
+
+	@BeforeEach
+	void setUp() {
+		중식 = new MenuGroup(1L, "중식");
+		탕수육 = new Product(1L, "탕수육", BigDecimal.valueOf(10000));
+		깐풍기 = new Product(2L, "깐풍기", BigDecimal.valueOf(12000));
+
+		탕수육중 = new MenuProduct(1L, 1L, 탕수육.getId(), 1);
+		깐풍기중 = new MenuProduct(2L, 1L, 깐풍기.getId(), 1);
+
+		menuProducts = new ArrayList<>();
+		menuProducts.add(탕수육중);
+		menuProducts.add(깐풍기중);
+
+		A세트 = new Menu(1L, "A세트", BigDecimal.valueOf(20000), 중식.getId(), menuProducts);
+		B세트 = new Menu(1L, "B세트", BigDecimal.valueOf(23000), 중식.getId(), menuProducts);
+	}
+
+	@DisplayName("Menu 생성을 테스트 - happy path")
+	@Test
+	void testCreateMenu() {
+		when(menuGroupDao.existsById(eq(A세트.getMenuGroupId()))).thenReturn(true);
+		when(productDao.findById(탕수육중.getProductId())).thenReturn(Optional.of(탕수육));
+		when(productDao.findById(깐풍기중.getProductId())).thenReturn(Optional.of(깐풍기));
+		when(menuDao.save(eq(A세트))).thenReturn(A세트);
+		when(menuProductDao.save(any())).thenReturn(탕수육중).thenReturn(깐풍기중);
+
+		Menu actual = menuService.create(A세트);
+
+		verify(menuProductDao, times(2)).save(any());
+
+		List<Long> actualMenuProductsId = actual.getMenuProducts()
+			.stream()
+			.map(MenuProduct::getProductId)
+			.collect(Collectors.toList());
+		List<Long> expectedMenuProductIds = A세트.getMenuProducts()
+			.stream()
+			.map(MenuProduct::getProductId)
+			.collect(Collectors.toList());
+		assertThat(actualMenuProductsId).containsExactlyElementsOf(expectedMenuProductIds);
+		assertThat(actual.getName()).isEqualTo(A세트.getName());
+	}
+
+	@DisplayName("메뉴 가격이 0보다 작은경우 오류발생")
+	@Test
+	void testCreateErrorPriceZero() {
+		Menu menu = new Menu(1L, "menu", BigDecimal.valueOf(-1), 1L, null);
```

Comment:
```suggestion
		BigDecimal price = BigDecimal.valueOf(-1);
		Menu menu = new Menu(1L, "menu", price, 1L, null);
```

가격을 변수로 따로 빼주면 어떨까요?